arches/app/templates/views/components/resource-report-abstract.htm

```diff
@@ -5,16 +5,7 @@
 <div class='loading-mask'></div>
 <!--/ko-->
 <!-- ko ifnot: loading() -->
-<div 
-    style="
-        display: flex;
-        flex-direction: column;
-        justify-content: space-between;
-    "
-    data-bind="
-        style: { height: $data.summary ? 'inherit' : 'calc(100vh - 50px)' }
```

Comment:
@phudson-he there is logic here to decide styling based on whether there is a summary property defined in the viewmodel.  You have removed it but not put it anywhere else.

If you have set the default class to include the `height: calc(100vh - 50px)` you may need to do something like

```html
data-bind="css: { height: $data.summary ? 'has-summary' : '' }

```

```css
resource-component-abstract-container has-summary{
   height: inherit;
}
```

Reply:
Before making changes, it may be a good idea to check what it looks like when a summary is present

Reply:
Change made in latest commit